docs/source/json-api/index.rst

```diff
@@ -178,8 +179,28 @@ For production use, we have a tool in development for generating proper
 RSA-encrypted tokens locally, which will arrive when the service also
 supports such tokens.
 
+Passing token with HTTP
+=======================
+
+Set HTTP header ``Authorization: Bearer copy-paste-token-here`` for
+normal requests.
+
+Passing token with WebSockets
+=============================
+
+WebSocket clients support a "subprotocols" argument; this is usually in
```

Comment:
The MDN docs of the WebSocket class speak of "protocols" instead of "subprotocols". Should we say something like
```suggestion
WebSocket clients support a "subprotocols" (sometimes simply called "protocols") argument; this is usually in
```

Reply:
50ce596438bcd3a53436ea5e1e232de1dd23b7ff ; thanks